public/util/data.ts

```diff
@@ -155,6 +155,20 @@ export function parseBinaryScoreResponse(res: BinaryScoreResponse): RankedSet {
   });
   return rankedSet;
 }
+// include the highlight
+export function getAllDataItems(includedActive: boolean): TableRow[] {
+  const tableData = includedActive
+    ? datasetGetters.getHighlightedIncludeTableDataItems(store)
+    : datasetGetters.getHighlightedExcludeTableDataItems(store);
+  const highlighted = tableData
+    ? tableData.map((h) => {
+        return { ...h, isExcluded: true };
```

Comment:
Could you explain with a comment why you need to add a new property `isExcluded`

Reply:
Thank you, that makes things so much clearer.